frontend/src/exams/generate-financial-report-modal.vue

```diff
@@ -16,47 +16,99 @@
+        <b-col sm="6">
+          <b-form-input id="startDate"
+                        type="date"
+                        v-model=startDate />
+          <b-form-invalid-feedback></b-form-invalid-feedback>
+        </b-col>
       </b-row>
       <b-row class="my-1">
         <b-col sm="4"><label>End Date:</label></b-col>
-        <b-col sm="6"><b-form-input id="endDate" type="date" v-model=endDate></b-form-input></b-col>
+        <b-col sm="6"><b-form-input id="endDate"
+                                    type="date"
+                                    v-model=endDate />
+        </b-col>
+      </b-row>
+      <b-row>
+        <b-col sm="4"><label>Exam Types:</label></b-col>
+        <b-col sm="6>">
+          <b-form-group>
+            <b-form-radio-group :options="options"
+                                stacked
+                                v-model="selectedExamType" />
+          </b-form-group>
+        </b-col>
+      </b-row>
+      <b-row>
+        <b-col sm="4"><label>Office:</label></b-col>
+        <b-col sm="6">
+          <OfficeDropDownFilter />
+        </b-col>
       </b-row>
     </b-container>
   </b-modal>
 </template>
 
 <script>
     import { mapState, mapMutations, mapActions } from 'vuex'
+    import OfficeDropDownFilter from './office-dropdown-filter'
+    import moment from 'moment'
     export default {
         name: "FinancialReportModal",
+        components: {
+          OfficeDropDownFilter,
+        },
+        mounted() {
+          this.getOffices()
+        },
         data() {
           return {
             startDate: '',
             endDate: '',
+            options: [
+              {text: 'ITA - Individual', value: 'ita_individual'},
+              {text: 'ITA - Group', value: 'ita_group'},
+              {text: 'Pesticide', value: 'pesticide'},
+              {text: 'Bulk Milk Tank Grader', value: 'milk_tank'}
+            ],
+            selectedExamType: '',
           }
         },
         methods: {
           ...mapActions([
-            'getExamsExport'
+            'getExamsExport',
+            'getExamTypes',
+            'getOffices'
           ]),
           ...mapMutations([
-            'toggleGenFinReport'
+            'toggleGenFinReport',
+            'setSelectedOffice'
           ]),
           submit() {
-
-            console.log("Pressed Submit")
             let form_start_date = this.startDate
             let form_end_date = this.endDate
-            let url= '/exams/export/?start_date=' + form_start_date + '&end_date=' + form_end_date
+            let exam_type = this.selectedExamType
+            let office_name = this.selectedOffice
+            let url= '/exams/export/?start_date=' + form_start_date + '&end_date=' + form_end_date + '&exam_type='
+                + exam_type + '&office_name=' + office_name
+            let today = moment().format('YYYY_MM_DD-HH:MM:SS')
```

Comment:
I believe colons are an invalid filename in windows. If we're doing ISO 8601 then something like `YYYY-MM-DDTHHMMSS` should work. Not sure how moment plays with that `T` - might need to do `YYYY-MM-DD_HHMMSS`.

Reply:
fixed.